### PR TITLE
Adding binder

### PR DESCRIPTION
**Pull request by PMSanelib**
Please review and merge

### CODE REVIEW

File: api/src/main/java/org/sanelib/ils/api/dto/binder/BinderDto.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
```
Comment (abhiptl) [inline api/src/main/java/org/sanelib/ils/api/dto/binder/BinderDto.java:129]: Please provide toString method in every dto that api exposes to client..

File: api/src/main/java/org/sanelib/ils/api/services/binder/BinderService.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        List viewList = binderViewRepository.getAllBinders();
+
+        dtoList.addAll((Collection) viewList.stream().map(v -> binderViewConverter.convert((BinderView) v)).collect(Collectors.toList()));
+
+        return dtoList;
+    }
+
+    @POST
+    public String addBinder(BinderDto dto) throws Throwable {
+        return execute(dto, ActivitiProcessConstants.Admin.ADD_BINDER);
```
Comment (abhiptl) [inline api/src/main/java/org/sanelib/ils/api/services/binder/BinderService.java:51]: BindingService is having compilation error. ActivitiProcessConstants.Admin.ADD_BINDER,UPDATE_BINDER,DELETE_BINDER are not added.

**Comment by PMSanelib** [comment]
Applying changes suggested and closing other PR.

**Comment by PMSanelib** [comment]
Cleanup and changes applied.

**Pull request merged**